### PR TITLE
Handle rate limit

### PR DESCRIPTION
**Pull request by sergeymirzoyan1991**
response:
```
{
    "ebs": {
        "success": [],
        "failure": [
            {
                "id": "vol-012ff9da6171e34d2",
                "reason": "The volume 'vol-012ff9da6171e34d2' does not exist.",
		"code": "InvalidVolume.NotFound"
            },
            {
                "id": "vol-012ff9da6171e34d4",
                "reason": "Request limit exceeded.",
		"code": "RequestLimitExceeded"
            }
        ]
    }
}
```

### CODE REVIEW

File: src/adapters/aws/clients/aws-client.ts [hunk 1]
```diff
@@ -35,22 +35,39 @@ export default class AwsClient {
...
-    // @ts-ignore
-    const result: {status: string, value: string, reason: string}[] = await Promise.allSettled(promises)
-    for (let i = 0; i < result.length; i++) {
-      if (result[i].status === 'fulfilled') {
-        response.addSuccess(result[i].value)
-      } else {
-        response.addFailure(new CleanFailureResponse(request.resources[i].id, result[i].reason))
+    if (hasRateLimitError) {
+      for (let i = start; i < request.resources.length; i++) {
+        response.addFailure(new CleanFailureResponse(request.resources[i].id, 'Request limit exceeded.', 'RequestLimitExceeded'))
```
Comment (marashlyansuren) [inline src/adapters/aws/clients/aws-client.ts:70]: @sergeymirzoyan1991 jan please use come constant error codes, general for engine, so we can use them for GCP as well.

Comment (marashlyansuren) [inline src/adapters/aws/clients/aws-client.ts:70]: @sergeymirzoyan1991 jan please use come constant error codes, general for engine, so we can use them for GCP as well.

Comment (sergeymirzoyan1991) [reply src/adapters/aws/clients/aws-client.ts]: done

Comment (sergeymirzoyan1991) [reply src/adapters/aws/clients/aws-client.ts]: done

**Review by tsardaryan — APPROVED** [review]

**Pull request merged**